src/display/api.js

```diff
@@ -883,42 +919,49 @@ class PDFPageProxy {
   }
 
   /**
-   * @return {number} Page number of the page. First page is 1.
+   * @type {number} Page number of the page. First page is 1.
```

Comment:
In the other files we still have `return`, so why is that changed to `type` here and below? It actually represents the return value of the function.

Reply:
The `.d.ts` was generated with no types when using `@return`, and as far as I could figure out, `get` properties should be documented like members using `@type`. In the current html API (draft) documentation you can see that the type is not shown https://mozilla.github.io/pdf.js/api/draft/PDFPageProxy.html#pageNumber

Here is a comparison
* Using `@return`
  ![image](https://user-images.githubusercontent.com/13413409/53595303-2f3de300-3b9d-11e9-96ed-4cada5daa168.png)

* Using `@type`
  ![image](https://user-images.githubusercontent.com/13413409/53595403-75934200-3b9d-11e9-8191-89b99970741a.png)

The reason why I did not change in all files was that i was focusing on `api.js`.

Reply:
Good point, I had no idea that there was actually a difference here. It's probably best to keep it as-is for this patch and create a follow-up issue to adjust the other places later.